ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -237,6 +237,9 @@ function generateFilter(type, property, value, metacardDefinitions) {
   if (!metacardDefinitions) {
     metacardDefinitions = require('../component/singletons/metacard-definitions.js')
   }
+  if (metacardDefinitions.metacardTypes[property] === undefined) {
+    return null
```

Comment:
is it safe to return null here? None of the callers will break?

Reply:
I believe that the only way to actually query on a property that is unknown/undefined is by using an imported search form, and I didn't notice any issues with that after making this change. Also, from what I can tell, `filter-builder` builds the cql using `CQLUtil`'s `transformFilterToCQL` function, which uses `cql.js`'s read/write functions that can handle null filters.